fineract-provider/src/main/java/org/apache/fineract/portfolio/client/service/ClientReadPlatformServiceImpl.java

```diff
@@ -559,11 +559,143 @@ public Collection<ClientData> retrieveActiveClientMembersOfCenter(final Long cen
                 hierarchySearchString, centerId, ClientStatus.ACTIVE.getValue());
     }
 
-    private static final class ClientMapper implements RowMapper<ClientData> {
+    private static final class ParentGroupsMapper implements RowMapper<GroupGeneralData> {
```

Comment:
Why has this (and the below classes) been reordered?

Reply:
Pretty simple, It was not reordered, ClientMapper was replaced with a mapper using MapStruct